index.html

```diff
@@ -64,84 +65,109 @@ <h4>CANOPY</h4>
                 <li class="skill-1">Javascript</li>
             </ul>
             <button class="btn"><a href="">See Project</a></button>
+    </div>   
         </article>
```

Comment:
- [ ] Dear @jerymey, For the buttons, when I hover over the button, I notice the color for the background and text becomes the same, which prevents me from reading the text while hovering over it. Please use the button interaction layout from figma and apply the sma effect.

### Your button on hover 
![image](https://github.com/jerymey/Portfolio-setup/assets/69077061/9a8118f0-9cc3-4b66-a450-f719170fe3b7)

### Figma layout 
![image](https://github.com/jerymey/Portfolio-setup/assets/69077061/e2131cd6-7019-4789-aec1-82d411e24390)